duckdb/join-duckdb.R

```diff
@@ -61,22 +63,31 @@ invisible({
 })
 
 if (!uses_NAs) {
+  if (on_disk) {
+    unlink('clean.db')
+    invisible(dbExecute(con, "attach 'clean.db'"))
+    db_name = "clean."
+  }
+  else {
+    db_name = ""
+  }
+
```

Comment:
I wonder why it is nested in `!uses_NA` branch?

Reply:
If there are no NA's, then there is an extra step during ingestion where we attach a new persistent db that has just the tables with enums. If there are NA's we just rename the tables and don't need to special logic. 

If I'm not mistaken duckdb can handle NA's in enums. I will update the script to reflect this when the report gets published